node/popn.go

```diff
@@ -772,7 +772,24 @@ func (nd *node) Load(ctx context.Context, args *GetArgs) (chan GetResult, error)
 				s = sel.Entries()
 			}
 
-			log.Info().Str("funds", filecoin.FIL(funds).String()).Msg("selected an offer")
+			bal, err := nd.exch.Wallet().Balance(ctx, nd.exch.Wallet().DefaultAddress())
+			if err != nil {
+				sendErr(err)
+				return
+			}
+
+			gasFee := filecoin.NewInt(100_000_000_000_000_000) // 0.1 FIL
+			fundsWithGasFee := filecoin.BigAdd(funds, gasFee)
+			estimatedBalanceAfterPayment := filecoin.BigSub(bal, fundsWithGasFee)
+
+			if bal.LessThan(fundsWithGasFee) {
+				insufficientFundsErr := fmt.Errorf("insufficient funds, at least %d attoFIL required but only %d available", fundsWithGasFee, bal)
+				sendErr(insufficientFundsErr)
+				return
+			}
+
+			fmt.Printf("==> Current Fil balance: %d attoFIL\n", bal)
```

Comment:
I think you can remove these. This method is mostly used via RPC so users would rarely read these logs unless the node crashes or something. Then we'd use `log.Debug()` to track things.